npm/pkg/dataplane/policies/policy_windows.go

```diff
@@ -100,9 +105,11 @@ func (acl *ACLPolicy) convertToAclSettings() (*NPMACLPolSettings, error) {
 	policySettings.LocalAddresses = srcListStr
 	policySettings.RemoteAddresses = dstListStr
 	policySettings.RemotePorts = dstPortStr
+	policySettings.LocalPorts = ""
 	if policySettings.Direction == hcn.DirectionTypeOut {
 		policySettings.LocalAddresses = dstListStr
 		policySettings.LocalPorts = dstPortStr
+		policySettings.RemotePorts = ""
```

Comment:
It looks unnecessary. policySettings has default value (empty string) for string.

Reply:
This is needed unlike the previous comment because the we are first setting the policy remote ports for In and then changing to out, so to override the previous assignment we are re-assigning this value to empty string.

Reply:
I guess it will be easier to understand to have `if else` based on `hcn.DirectionTypeOut`
```go
if policySettings.Direction == hcn.DirectionTypeOut {
		policySettings.LocalAddresses = dstListStr
		policySettings.RemoteAddresses = srcListStr
                policySettings.LocalPorts = dstPortStr
}else  { // maybe In?
	policySettings.LocalAddresses = srcListStr
	policySettings.RemoteAddresses = dstListStr
	policySettings.RemotePorts = dstPortStr
}
```

If I missed something, stay with current code, but slightly adjust order in `if` condition to have the same order above assignment since it helps understand what wat changed.
```go
	policySettings.LocalAddresses = srcListStr
	policySettings.RemoteAddresses = dstListStr
	policySettings.RemotePorts = dstPortStr
	policySettings.LocalPorts = ""
	if policySettings.Direction == hcn.DirectionTypeOut {
		policySettings.LocalAddresses = dstListStr
		policySettings.RemoteAddresses = srcListStr
		policySettings.RemotePorts = ""
                policySettings.LocalPorts = dstPortStr
	}

```

Reply:
But do you see a huge readability issue with the way it is currently ? I feel like by explicitly assigning to "" it is clear what to expect from a ACL policy in a given direction and removes any assumptions ? wdyt ?